packages/app-bridge/src/types/Color.ts

```diff
@@ -134,4 +127,33 @@ export type ColorCreate = {
     alpha: number;
 };
 
-export type ColorPatch = Partial<Omit<Color, 'id' | 'hex' | 'hue' | 'saturation' | 'lightness'>>;
+export type ColorPatch = Partial<Omit<ColorRevision, 'hex' | 'hue' | 'saturation' | 'lightness'>> & {
+    name?: Nullable<string>;
+    sort?: number;
+};
+
+type Rgba = {
+    red: Nullable<number>;
+    green: Nullable<number>;
+    blue: Nullable<number>;
+    alpha: Nullable<number>;
+};
+
+type Cmyk = {
+    c: Nullable<number>;
+    m: Nullable<number>;
+    y: Nullable<number>;
+    k: Nullable<number>;
```

Comment:
Q: should these keys be written in extensive (cyan, magenta, yellow, black) to be consistent with Rgb?

Reply:
Yes adjusted it